applications/dashboard/design/admin.css

```diff
@@ -7911,15 +7911,15 @@ label[id^=iCheck] {
 
 .modal-body {
   position: relative;
-  max-height: calc(94vh - (2 * 48px));
+  max-height: calc(92vh - 108px);
   overflow-y: auto;
   padding: 0 18px;
 }
 .modal-confirm .modal-body {
   padding: 18px;
 }
 .modal-full-height .modal-body {
-  height: calc(90vh - (2 * 48px));
+  height: calc(92vh - 108px);
```

Comment:
Just for the record. 
`2*48 = 96      !=     108`
Is it intentional?

Reply:
Yup. It's weird. Not sure why, but I don't want to spend more time in our old modal.

We use a new one with better calculations for everything going forward.